server/routes/api/collections/collections.ts

```diff
@@ -1008,12 +1004,9 @@ router.post(
       method: ["withMembership", user.id],
     }).findByPk(id, {
       transaction,
+      rejectOnEmpty: true,
```

Comment:
I think we might need to revert this back. It's kind of [problematic](https://github.com/outline/outline/issues/6207) with the `"withMembership"` scope.

Reply:
The behavior is overridden in Collection model, same as Document to get around this bug :)